src/Auth/UserRegistration.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Copyright (C) 2018 Adshares sp. z. o.o.
```

Comment:
`z.` should not have `.` :)

Reply:
Yep, Guess from which config I copied copyrights :)

Reply:
I will change all copyrights in a seperate branch